db/schema.rb

```diff
@@ -191,13 +191,13 @@
   create_table "usps_confirmation_codes", force: :cascade do |t|
     t.integer "profile_id", null: false
     t.string "otp_fingerprint", null: false
-    t.datetime "code_sent_at", default: -> { "now()" }, null: false
+    t.datetime "code_sent_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["profile_id"], name: "index_usps_confirmation_codes_on_profile_id"
   end
 
-  create_table "usps_confirmations", force: :cascade do |t|
+  create_table "usps_confirmations", id: :serial, force: :cascade do |t|
     t.text "entry", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
```

Comment:
Where did these schema changes come from?

Reply:
I haven't confirmed, but my guess is from rerunning the migrations since gems have been updated. With the new changes, this file has been reverted.

FWIW: `serial` is the default postgres is column type, so that shouldn't change anything, but the change in default datetime value is a mystery to me.